src/static/js/demo.js

```diff
@@ -96,9 +99,53 @@ function configureControlPanel() {
   }
 }
 
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+function preventZoomingPage() {
```

Comment:
:question: can we add a short comment to explain why we want this? You explain it to me and I have already forgotten :smile:

Reply:
thanks for the comment